src/fu-util-common.c

```diff
@@ -1304,6 +1304,55 @@ fu_util_update_state_to_string(FwupdUpdateState update_state)
 	return NULL;
 }
 
+static gchar *
+fu_util_device_problem_to_string(FwupdClient *client, FwupdDevice *dev, FwupdDeviceProblem problem)
+{
+	if (problem == FWUPD_DEVICE_PROBLEM_NONE)
+		return NULL;
+	if (problem == FWUPD_DEVICE_PROBLEM_UNKNOWN)
+		return NULL;
+	if (problem == FWUPD_DEVICE_PROBLEM_SYSTEM_POWER_TOO_LOW) {
+		if (fwupd_client_get_battery_level(client) == FWUPD_BATTERY_LEVEL_INVALID ||
+		    fwupd_client_get_battery_threshold(client) == FWUPD_BATTERY_LEVEL_INVALID) {
+			/* TRANSLATORS: as in laptop battery power */
+			return g_strdup(_("System power is too low to perform the update"));
+		}
+		return g_strdup_printf(
+		    /* TRANSLATORS: as in laptop battery power */
+		    _("System power is too low to perform the update (%u%%, requires %u%%)"),
+		    fwupd_client_get_battery_level(client),
+		    fwupd_client_get_battery_threshold(client));
+	}
+	if (problem == FWUPD_DEVICE_PROBLEM_UNREACHABLE) {
+		/* TRANSLATORS: for example, a BlueTooth mouse that is in powersave mode */
```

Comment:
I don't think Bluetooth needs the uppercase T

Reply:
Indeed, all three fixed. thanks.